crates/common/src/compile.rs

```diff
@@ -606,21 +606,20 @@ impl Display for SizeReport {
                 ((runtime_margin, runtime_color), (init_margin, init_color))
             } else {
                 let runtime_margin =
-                    CONTRACT_RUNTIME_SIZE_LIMIT as isize - contract.runtime_size as isize;
-                let init_margin =
-                    CONTRACT_INITCODE_SIZE_LIMIT as isize - contract.init_size as isize;
+                CONTRACT_RUNTIME_SIZE_LIMIT as isize - contract.runtime_size as isize;
+            let init_margin = CONTRACT_INITCODE_SIZE_LIMIT as isize - contract.init_size as isize;
```

Comment:
Seems this changes only involve indentation and the previous one looks more accurate to me, not sure how this happened.